xray/services/utils/ignorerulebody.go

```diff
@@ -0,0 +1,69 @@
+package utils
+
+import "time"
+
+type IgnoreRuleParams struct {
+	Notes         string        `json:"notes"`
+	ExpiresAt     time.Time     `json:"expires_at,omitempty"`
+	IgnoreFilters IgnoreFilters `json:"ignore_filters"`
+}
+
+type IgnoreRuleBody struct {
+	Id        string    `json:"id,omitempty"`
+	Author    string    `json:"author,omitempty"`
+	Created   time.Time `json:"created,omitempty"`
+	IsExpired bool      `json:"is_expired,omitempty"`
+	IgnoreRuleParams
+}
+
+type IgnoreFilters struct {
+	Vulnerabilities  []string                      `json:"vulnerabilities,omitempty"`
+	Licenses         []string                      `json:"licenses,omitempty"`
+	CVEs             []string                      `json:"cves,omitempty"`
+	Policies         []string                      `json:"policies,omitempty"`
+	Watches          []string                      `json:"watches,omitempty"`
+	DockerLayers     []string                      `json:"docker-layers,omitempty"`
+	OperationalRisks []string                      `json:"operational_risk,omitempty"`
+	Exposures        []ExposuresFilterName         `json:"exposures,omitempty"`
+	ReleaseBundles   []IgnoreFilterNameVersion     `json:"release-bundles,omitempty"`
```

Comment:
```suggestion
	ReleaseBundles   []IgnoreFilterNameVersion     `json:"release_bundles,omitempty"`
```
please use the same type of format cases and don't mix (you have snake-case and kebab-case, or is it what the API is expecting?

Reply:
This is what the API expects. It failed the test I added on release bundle.
See [here](https://jfrog.com/help/r/xray-rest-apis/create-ignore-rule)  operational_risk is snake and release-bundles and docker-layers is kebab